### PR TITLE
Feature create event button saves to db

### LINKED ISSUE #55: Add an Event Functionality (UI)
As a Software Developer I will want to create the UI for add an Event functionality.

Is your feature request related to a problem? Please describe.
Users need a user interface to add Events to their page.

Describe the solution you'd like
Create a user interface for adding an event.

Story Points
4-10

### PR DESCRIPTION
**Pull request by theGaryLarson**
As a frontend developer, I connected the handleSubmit event of the Create Event button to the nest.js backend.

Required
- [x] enabling cors
- [x] creating a DTO for the event/activity
- [x] defining the method createActivity
- [x] calling the method in handleSubmit

resolves #55

### CODE REVIEW

File: app/create-event/page.tsx [hunk 1]
```diff
@@ -37,10 +37,33 @@ const CreateEvent = () => {
...
         e.preventDefault();
-        // Save the event in the db through nest.js
         console.log(eventData);
+        createActivity(eventData);
+
     };
 
+    const createActivity = async (activityData: any) => {
+        try {
+            const response = await fetch('http://localhost:3000/api/activity/add', {
```
Comment (bcko) [inline app/create-event/page.tsx:47]: Looks great. Let's also create a ticket that would explore below client libraries (SWR, React Query) to fetch data. 

https://nextjs.org/docs/app/building-your-application/data-fetching/fetching-caching-and-revalidating#fetching-data-on-the-client-with-third-party-libraries
https://swr.vercel.app/
https://tanstack.com/query/v3/

**Review by bcko — APPROVED** [review]

File: app/create-event/page.tsx (hunk 1 above)
Comment (theGaryLarson) [reply app/create-event/page.tsx:47]: I included these SWR and React Query links in my google docs task list, which I posted in Slack.

**Review by brinkbrink — APPROVED** [review]
LGTM but I'm curious about some syntax--I remember somebody mentioning that we are moving away from using "event" and instead using "activity." Has a ticket been made for auditing this? 

Should there be an issue (maybe for fall)? I definitely agree that using "event" is confusing and that change should be made throughout the codebase. I'll add that to the fall planning Miro for now so we don't forget. Feel free to delete if it's already been acknowledged elsewhere (I admit that I don't follow the NSC repos quite as closely as Belinda's).

**Review by taylorpapke — APPROVED** [review]
LGTM

**Comment by theGaryLarson** [comment]
@brinkbrink My thoughts were to use Activity within code but have routes use the word Event. That way when a URL is displayed in the browser an user sees the word event. Creating a ticket to audit this is a great idea.

**Pull request merged**